workflow/rules/bwa-mem.smk

```diff
@@ -0,0 +1,48 @@
+# vim: syntax=python tabstop=4 expandtab
+# coding: utf-8
+
+__author__ = "Jonas Almlöf"
+__copyright__ = "Copyright 2021, Jonas Almlöf"
+__email__ = "jonas.almlof@scilifelab.uu.se"
+__license__ = "GPL-3"
+
+
+rule bwa_mem:
+    input:
+        fastq=["Pre-alignment/merge/{sample}_R1.fq.gz", "Pre-alignment/merge/{sample}_R2.fq.gz"],
+    output:
+        bam="Alignment/bwa-mem/{sample}.bam",
+    log:
+        log="Logs/Aligment/bwa-mem/{sample}.log",
+    params:
+        index=config["references"]["genome"],
```

Comment:
Files should be part of input to have snakemake check that they are indeed available

Reply:
Also how it is specified in the wrapper

Reply:
If we want to force snakemake to check for it, we could add as an input.